src/src/tx_main.cpp

```diff
@@ -404,7 +404,20 @@ uint8_t adjustPacketRateForBaud(uint8_t rateIndex)
     if (crsf.GetCurrentBaudRate() == 115200) {
       while (rateIndex < RATE_MAX) {
         expresslrs_mod_settings_s const * const ModParams = get_elrs_airRateConfig(rateIndex);
-        if (ModParams->enum_rate <= RATE_LORA_250HZ) {
+        uint32_t RCpacketInterval = ModParams->interval * ModParams->numOfSends;
+        if (RCpacketInterval >= 4000) {
+          break;
+        }
+        rateIndex++;
+      }
+    }
+    else
```

Comment:
I'd add { } to the block otherwise it's hard to read

Reply:
Hows it look now?

Im also happy if you would like to make the changes directly if Ive misunderstood.